k8s/wordpress/chart/wordpress/templates/wordpress-manifests.yaml

```diff
@@ -79,6 +79,10 @@ spec:
             secretKeyRef:
               name: {{ .Release.Name }}-wordpress-secret
               key: wp-user
+        resources:
```

Comment:
Build fails due to wrong section, please move it before `env` section

Reply:
Moved, re-running build now.